tests/kernel/iop/thread/create.c

```diff
@@ -0,0 +1,92 @@
+#include <common-iop.h>
+#include <thbase.h>
+
+void nullThreadProc(u32 param) {
+	//Do nothing
+}
+
+void printThreadInfo(s32 threadId) {
+	iop_thread_info_t threadStat;
+	memset(&threadStat, 0, sizeof(iop_thread_info_t));
+	s32 result = ReferThreadStatus(threadId, &threadStat);
+	if (result >= 0) {
+		printf("attr: %08x, option: %08x, stack size: %08x, init prio: %02x, curr prio: %02x, status: %02x, ", 
+			threadStat.attr, threadStat.option, threadStat.stackSize, 
+			threadStat.initPriority, threadStat.currentPriority, threadStat.status);
+		if(threadStat.entry == &nullThreadProc) {
+			printf("entry: &nullThreadProc");
+		} else {
+			printf("entry: %08x", threadStat.entry);
+		}
+		printf("\n");
+	} else {
+		printf("refer failed -> result: %d\n", result);
+	}
+}
+
+
+void createThread(s32 attr, void *entry, s32 prio, s32 stackSize, u32 option) {
+	iop_thread_t threadParam;
+	memset(&threadParam, 0, sizeof(iop_thread_t));
+	threadParam.attr      = attr;
+	threadParam.thread    = entry;
+	threadParam.priority  = prio;
+	threadParam.stacksize = stackSize;
+	threadParam.option    = option;
+	
+	s32 thId = CreateThread(&threadParam);
+	
+	if(thId > 0) {
+		printf("  succeeded -> ");
+		printThreadInfo(thId);
+		DeleteThread(thId);
+	} else {
+		printf("  failed -> result: %d\n", thId);
+	}
+}
+
+int _start(int argc, char **argv) {
+	printf("-- TEST BEGIN\n");
+	
+	printf("valid:\n");
+	createThread(TH_C, &nullThreadProc, 10, 0x800, 0x01234567);
```

Comment:
If I create a thread, delete it, and then create another thread, will the stack be positioned in the same place?  Or does it use a rolling allocation algorithm?

-[Unknown]